scripts/common.py

```diff
@@ -0,0 +1,67 @@
+import json
+import os
+from pathlib import Path
+
+COOKIE_CUTTER_PROJECT_NAMES = [
+    "{{cookiecutter.project_name}}",
+    "{{ cookiecutter.project_name }}",
+    "{{ cookiecutter.project_slug }}",
+]
+
+NON_RUNTIME_DIRECTORY = [
+    "tests",
+    "scripts",
+    "buildspecs",
+    ".github",
+    ".git",
+    ".pytest_cache",
+]
+
+
+def find_cookie_cutter_projects(runtime_directories):
+    projects = []
+    def _traverse(directory):
+        directory_files = os.listdir(directory)
+        name = _get_project_name(directory_files)
+        if name:
+            projects.append((directory, name))
+            return
+
+        for file in directory_files:
+            file_path = Path(directory, file)
+            if Path.is_dir(file_path):
```

Comment:
we may need to check for symlinks ordinarily to avoid loops but we do control the source code here, so should be fine.

Reply:
Good call-out. I didn't consider that when writing this but we should be okay here. Definitely something to keep in mind going forward though